test/new/db/archos/linux-x64/dbg_bps

```diff
@@ -197,23 +197,27 @@ EOF
 RUN
 
 NAME='valid db'
-FILE=../bins/elf/analysis/x86-helloworld-gcc
+FILE=../bins/elf/analysis/x64-loop
 ARGS=-d
 CMDS=<<EOF
-db 0x0
-db~1[12];
-EXPECT=<<RUN
-"invalid"
+e scr.color=0
+db main
+db~1[8]
+EOF
+EXPECT=<<EOF
+valid
 EOF
 RUN
 
 NAME='invalid db'
-FILE=../bins/elf/analysis/x86-helloworld-gcc
+FILE=../bins/elf/analysis/x64-loop
 ARGS=-d
 CMDS=<<EOF
-db entry0
-db~1[12]
-EXPECT=<<RUN
-"valid"
+e scr.color=0
```

Comment:
I do t see the point on setting a breakpoint and not running the code to hit it

Reply:
I am only testing that the fields are properly updated, you will never hit an invalid breakpoint anyways and we have tests for valid breakpoints.

Reply:
the output matters imho